### ISSUE TITLE
Teaser selection has fixed limit of 10

### ISSUE THREAD

**Issue by matthijsch** [sulu/sulu#4331]
| Q | A
| --- | ---
| Bug? | yes
| New Feature? | no
| Sulu Version | <= 1.6.23
| Browser Version | not relevant

#### Actual Behavior

When selecting more than 10 pages in a teaser_selection block, only 10 pages are displayed. This is probably because the default limit of elasticsearch is 10

#### Expected Behavior

Expected is that selecting more than 10 pages every selected page is displayed

#### Steps to Reproduce

Create a teaser_selection block and add more than 10 pages

#### Possible Solutions

Set limit here:

https://github.com/sulu/sulu/blob/1.6.23/src/Sulu/Bundle/ContentBundle/Teaser/ContentTeaserProvider.php#L61

so this block would be something like:

```
$searchResult = $this->searchManager
            ->createSearch(implode(' OR ', $statements))
            ->setLimit(count($ids))
            ->indexes($this->getPageIndexes())
            ->locale($locale)
            ->execute();
```

**Comment by danrot** [comment]
Just tested it with ZendSearch, there this error does not happen... Is it maybe exclusively to ElasticSearch? Gonna test it tomorrow.

**Comment by danrot** [comment]
Fixed in #4998
